birdy/client/base.py

```diff
@@ -214,8 +216,8 @@ def _execute(self, pid, **kwargs):
             raise
 
         # Output type conversion
-        output = namedtuple(pid, [o.identifier for o in resp.processOutputs])
-        return output(**{o.identifier: self._process_output(o, pid) for o in resp.processOutputs})
+        Output = namedtuple('Output', [make_identifier(o.identifier) for o in resp.processOutputs])
+        return Output._make([self._process_output(o, pid) for o in resp.processOutputs])
```

Comment:
I think you could just use `Output(*[...])`, no ?

Reply:
Yes ... copy+paste form the docs:
https://docs.python.org/3/library/collections.html#collections.somenamedtuple._make

So, I thought to take the example code ...

Reply:
accept?

Reply:
I'd prefer using the public method.